### PR TITLE
Use 'add-on' instead of 'addon'

### LINKED ISSUE #40: Use "Add-ons" instead of "addons"
This is a request from @jvillalobos

### PR DESCRIPTION
**Pull request by magopian**
Fixes #40

r? @n1k0 @natim

### CODE REVIEW

File: amo-blocklist.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "addons": {
-    "name": "addons",
+  "add-ons": {
```
Comment (Natim) [inline amo-blocklist.json]: This means we need to rename the collection too

Comment (magopian) [inline amo-blocklist.json]: Indeed. Is that an issue? We "just" need to change that in xml2kinto

Comment (Natim) [inline amo-blocklist.json]: Yes no worries.

**Comment by n1k0** [comment]
LGTM, r+

File: amo-blocklist.json (hunk 1 above)
Comment (Natim) [inline amo-blocklist.json]: According to @leplatrem we also need to change that in Gecko...

Comment (jvillalobos) [inline amo-blocklist.json]: The request in #40 was to change the label in the UI, not changing what we use in the blocklist output.

Comment (leplatrem) [inline amo-blocklist.json]: Yeah, I created for this https://github.com/Kinto/kinto-admin/issues/40

**Pull request merged**